src/js/themes/base.d.ts

```diff
@@ -935,6 +935,7 @@ export interface ThemeType {
         color?: ColorType;
       };
     };
+    focusIndicator?: boolean;
```

Comment:
Seems a little weird to have a `focus` object and then separately have `focusIndicator`

Reply:
yeah maybe put focusIndicator in the focus object or open to other ideas or naming